qgepqwat2ili/utils/ili2db.py

```diff
@@ -97,8 +97,9 @@ def import_xtf_data(schema, xtf_file, log_path):
     )
 
 
-def export_xtf_data(schema, model_name, xtf_file, log_path):
+def export_xtf_data(schema, model_name, export_model_name, xtf_file, log_path):
```

Comment:
That change I made before you completly change export_xtf_data function with you pull request [omit empty pg connection settings
#97](https://github.com/QGEP/qgepqwat2ili/pull/97/files#diff-1e13712646c26ed716e37e0cb03d731ca56457b8c516fd82f1548723b0811e67)

So we would have to add the variable --exportModels to your new version to be able to make such a call again:

> https://github.com/QGEP/qgepqwat2ili/issues/66#issuecomment-1305425590
> I tested with the following command, if an export of `SIA405_ABWASSER_2015_LV95` out of the `pg2ili_abwasser `schema is possible:
> 
> `java -jar C:/Daten/Dropbox/in_Arbeit/QGEP/Interlis_Schnittstelle_ili2pg/ili2pg-4.9.0/ili2pg-4.9.0.jar --export --modeldir %ilimodelpath%  --models SIA405_ABWASSER_2015_LV95 --exportModels SIA405_ABWASSER_2015_LV95 --skipReferenceErrors --sqlEnableNull --createEnumTabs --createFk --noSmartMapping --defaultSrsAuth EPSG --defaultSrsCode 2056 --dbhost %host% --dbport %port% --dbdatabase %db% --dbschema pg2ili_abwasser --dbusr %user% --dbpwd %pwd% --log  casse_SIA405_Abwasser_from_KEK.log --trace casse_SIA405_Abwasser_from_KEK.xtf`
> 
> In case of exporting SIA405_ABWASSER_LV95 the existing code has to be modified or a second export statement has to be added using these parameters: `--models SIA405_ABWASSER_2015_LV95 --exportModels SIA405_ABWASSER_2015_LV95`